Ginger/Ginger/Actions/ActionConversion/ActionConverterPage.xaml.cs

```diff
@@ -315,6 +318,7 @@ private void btnConvertAction_Click(object sender, RoutedEventArgs e)
                         }
                     }
                 }
+                Mouse.OverrideCursor = null;
             }
```

Comment:
please do Mouse.OverrideCursor = null; inside Finally block so it will be done even if exception will happen in middle of code (so mouse won't get stuck with Processing icon).
for showing Process is running better to show our 'Processing" imageMaker image, see sample in POM wizard when we learn the object so we show the rotatating processing image to make clear we running in backend.